libs/ballot-interpreter-nh-next/src/interpret.rs

```diff
@@ -0,0 +1,385 @@
+            },
+            geometry,
+        });
+    }
+
+    Ok(LoadedBallotPage {
+        ballot_image: LoadedBallotImage {
+            image,
+            border_inset,
+        },
+        geometry,
+    })
+}
+
+#[time]
+pub fn interpret_ballot_card(side_a_path: &Path, side_b_path: &Path, options: &Options) -> Result {
+    let LoadedBallotCard {
+        side_a,
+        side_b,
+        geometry,
+    } = load_ballot_card_images(side_a_path, side_b_path)?;
+
+    let mut side_a_debug = if options.debug {
+        ImageDebugWriter::new(side_a_path.to_path_buf(), side_a.image.clone())
+    } else {
+        ImageDebugWriter::disabled()
+    };
+    let mut side_b_debug = if options.debug {
+        ImageDebugWriter::new(side_b_path.to_path_buf(), side_b.image.clone())
+    } else {
+        ImageDebugWriter::disabled()
+    };
+
+    let (side_a_result, side_b_result) = rayon::join(
+        || {
+            find_timing_mark_grid(
+                side_a_path,
+                &geometry,
+                &side_a.image,
+                side_a.border_inset,
+                &mut side_a_debug,
+            )
+        },
+        || {
+            find_timing_mark_grid(
+                side_b_path,
+                &geometry,
+                &side_b.image,
+                side_b.border_inset,
+                &mut side_b_debug,
+            )
+        },
+    );
+
+    let (side_a_grid, side_a_normalized_img) = side_a_result?;
+    let (side_b_grid, side_b_normalized_img) = side_b_result?;
+
+    let ((front_image, front_grid, front_debug), (back_image, back_grid, back_debug)) =
+        match (&side_a_grid.metadata, &side_b_grid.metadata) {
+            (BallotPageMetadata::Front(_), BallotPageMetadata::Back(_)) => (
+                (
+                    side_a_normalized_img.unwrap_or(side_a.image),
+                    side_a_grid,
+                    side_a_debug,
+                ),
+                (
+                    side_b_normalized_img.unwrap_or(side_b.image),
+                    side_b_grid,
+                    side_b_debug,
+                ),
+            ),
+            (BallotPageMetadata::Back(_), BallotPageMetadata::Front(_)) => (
+                (
+                    side_b_normalized_img.unwrap_or(side_b.image),
+                    side_b_grid,
+                    side_b_debug,
+                ),
+                (
+                    side_a_normalized_img.unwrap_or(side_a.image),
+                    side_a_grid,
+                    side_a_debug,
+                ),
+            ),
+            _ => {
+                return Err(Error::InvalidCardMetadata {
+                    side_a: side_a_grid.metadata,
+                    side_b: side_b_grid.metadata,
+                })
+            }
+        };
+
+    let ballot_style_id = match &front_grid.metadata {
+        BallotPageMetadata::Front(metadata) => {
+            BallotStyleId::from(format!("card-number-{}", metadata.card_number))
```

Comment:
Since it's only a short-term need for NH timing mark encoded ballot metadata, I'm less inclined to make a change to `Election` to support this. Can live with a little duplication